schemas/Quote.json

```diff
@@ -6,15 +6,15 @@
   "properties": {
     "source_connector_account": {
       "description": "the connector's account on the source ledger",
-      "$ref": "Iri.json"
+      "type": "string"
     },
     "source_ledger": {
-      "description": "The URI of the source ledger",
-      "$ref": "Iri.json"
+      "description": "The ILP Address of the source ledger",
+      "type": "string"
```

Comment:
I think this should reference a proper schema (https://github.com/interledger/five-bells-shared/pull/150/files#r71853250)